src/components/Clock/Clock.tsx

```diff
@@ -12,51 +16,43 @@ enum ClockHand {
   ss,
 }
 
-const getDegreeByDate = (time: Date, hande: ClockHand): number => {
+const getDegreeByDate = (m: any, hande: ClockHand): number => {
```

Comment:
```suggestion
const getDegreeByDate = (m: Moment, hande: ClockHand): number => {
```

You can import Moment type form 'moment'